File: MSRstdlogret/MSRstdlogret.m
% ---------------------------------------------------------------------
% Book:         
% ---------------------------------------------------------------------
% Quantlet:     MSRstdlogret
% ---------------------------------------------------------------------
% Description:  MSRtaildax plots the standardized log-returns of Bayer, 
%               BMW, Siemens and Volkswagen.
% ---------------------------------------------------------------------
% Usage:        MSRstdlogret
% ---------------------------------------------------------------------
% Inputs:       None
% ---------------------------------------------------------------------
% Output:       Plot of the standardized log-returns.
% ---------------------------------------------------------------------
% Example:     
% ---------------------------------------------------------------------
% Author:       Wolfgang Härdle, Barbara Choros
% ---------------------------------------------------------------------

x1=load('Bay9906_standLogRet.dat');
x2=load('BMW9906_standLogRet.dat');
x3=load('Sie9906_standLogRet.dat');
x4=load('Vow9906_standLogRet.dat');

figure(1);plot(x1);ylim([-6 8]);title('Daily Standardized Log-Returns of Bayer');
figure(2);plot(x2);ylim([-6 8]);title('Daily Standardized Log-Returns of BMW');
figure(3);plot(x3);ylim([-6 8]);title('Daily Standardized Log-Returns of Siemens');
figure(4);plot(x4);ylim([-6 8]);title('Daily Standardized Log-Returns of Volkswagen');

File: MSRstdlogret/MSRstdlogret.R
# ---------------------------------------------------------------------
# Book:         
# ---------------------------------------------------------------------
# Quantlet:     MSRstdlogret
# ---------------------------------------------------------------------
# Description:  MSRstdlogret plots the standardized log-returns of Bayer, 
#               BMW, Siemens and Volkswagen.
# ---------------------------------------------------------------------
# Usage:        MSRstdlogret
# ---------------------------------------------------------------------
# Inputs:       None
# ---------------------------------------------------------------------
# Output:       Plot of the standardized log-returns.
# ---------------------------------------------------------------------
# Example:     
# ---------------------------------------------------------------------
# Author:       Zografia Anastasiadou 20110606
# ---------------------------------------------------------------------

graphics.off()
rm(list = ls(all = TRUE))
#setwd("C:/...")

x1 = read.table("Bay9906_standLogRet.dat")
x2 = read.table("BMW9906_standLogRet.dat")
x3 = read.table("Sie9906_standLogRet.dat")
x4 = read.table("Vow9906_standLogRet.dat")

plot(x1[, 1], ylim = c(-6, 8), main = "Daily Standardized Log-Returns of Bayer",
    type = "l", col = "blue", xlab = "", ylab = "")
plot(x2[, 1], ylim = c(-6, 8), main = "Daily Standardized Log-Returns of BMW",
    type = "l", col = "blue", xlab = "", ylab = "")
plot(x3[, 1], ylim = c(-6, 8), main = "Daily Log-Returns of Siemens",
    type = "l", col = "blue", xlab = "", ylab = "")
plot(x4[, 1], ylim = c(-6, 8), main = "Daily Standardized Log-Returns of Volkswagen",
    type = "l", col = "blue", xlab = "", ylab = "")